package.json

```diff
@@ -103,19 +103,37 @@
         "subPermissions": [
           "module.data-export.enabled"
         ],
+        "visible": false
+      },
+      {
+        "permissionName": "ui-data-export.view",
+        "displayName": "Data export: Can view only",
+        "subPermissions": [
+          "module.data-export.enabled",
+          "data-export.related-users.collection.get",
+          "data-export.file-definitions.item.post",
+          "data-export.file-definitions.upload.post",
+          "data-export.job-executions.items.download.get",
+          "data-export.job-executions.collection.get",
+          "data-export.logs.collection.get",
+          "data-export.mapping-profiles.collection.get",
+          "data-export.job-profiles.collection.get",
+          "data-export.mapping-profiles.item.get",
+          "data-export.export.post"
```

Comment:
Should we have post subPermissions in view permission?
"data-export.file-definitions.item.post",
"data-export.file-definitions.upload.post",
"data-export.export.post"